easy_rec/python/layers/input_layer.py

```diff
@@ -140,7 +158,7 @@ def __call__(self, features, group_name, is_combine=True, is_dict=False):
           embedding_reg_lst.append(tmp_embedding)
       regularizers.apply_regularization(
           self._embedding_regularizer, weights_list=embedding_reg_lst)
-      return seq_features
+      return seq_features, output_features, group_features
```

Comment:
this interface is already too complicated, consider refactor it into a new function